rotkehlchen/exchanges/bitstamp.py

```diff
@@ -299,7 +305,7 @@ def query_online_deposits_withdrawals(
         for idx in sorted(indices_to_delete, reverse=True):
             del crypto_asset_movements[idx]  # remove the crypto asset movements whose data we matched in the DB  # noqa: E501
 
-        return asset_movements + crypto_asset_movements
+        return asset_movements
```

Comment:
I think this log from when debugging would be useful to have here for all users. It would show us if there is another failure of matching and for which crypto transactions that was

```suggestion
        log.debug(f'Remaining Bitstramp unmatched {crypto_asset_movements=}')
        return asset_movements
```

Reply:
Fixed